app/config/question-bank.js

```diff
@@ -3324,7 +3333,7 @@ const questionBank = {
             values: [{
               heading: 'Funding priorities',
               content: [{
-                para: `RPA want to fund multi-tier systems that have 3 tiers or fewer directly above each other, to reduce the risk of keel bone fractures from collisions and falls.`
+                para: `RPA want to fund aviary systems that have 3 tiers or fewer directly above each other, to reduce the risk of keel bone fractures from collisions and falls.`
```

Comment:
pretty sure this needs to be dynamic (multi-tier for hen, aviary for pullet)

Reply:
done